.github/workflows/workflow.yml

```diff
@@ -178,27 +177,53 @@ jobs:
         echo "${{ secrets.DOCKER_PRIVATE_KEY }}" > ~/.ssh/id
         echo "${{ secrets.DOCKER_KNOWN_HOSTS }}" > ~/.ssh/known_hosts
         ssh linkerd-docker docker version
-    - name: Docker build
-      if: github.event_name != 'pull_request' || !github.event.pull_request.head.repo.fork
-      env:
-        DOCKER_HOST: ssh://linkerd-docker
-      run: |
-        export PATH="`pwd`/bin:$PATH"
-        DOCKER_TRACE=1 bin/docker-build
+        echo ::set-env name=DOCKER_HOST::ssh://linkerd-docker
+    - name: Try to load cached Go modules
+      if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork
+      uses: actions/cache@v1
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+    - name: Build docker images
+      run: DOCKER_TRACE=1 bin/docker-build
```

Comment:
> 2\. `docker-build will product a `cli-bin`image for us, we should use that rather than rely on`target/cli/linux/linkerd`.

@siggy From my understanding, `bin/docker-build` which calls `bin/docker-build-cli-bin` is already copying the CLI binary from the docker image into `target/cli/linux/linkerd`

Reply:
That's fair. My comment was coming from an overabundance of caution, in the unlikely case where a cached or incorrect cli ended up at `target/cli/linux/linkerd`, but I guess we should trust our own build scripts. ;)